aviatrix/resource_aviatrix_transit_external_device_conn.go

```diff
@@ -678,7 +700,11 @@ func resourceAviatrixTransitExternalDeviceConnRead(d *schema.ResourceData, meta
 	}
 
 	if conn != nil {
-		d.Set("vpc_id", conn.VpcID)
+		if strings.Contains(conn.VpcID, "~-~") {
+			d.Set("vpc_id", strings.Split(conn.VpcID, "~-~")[0])
+		} else {
+			d.Set("vpc_id", conn.VpcID)
+		}
```

Comment:
Won't this cause a diff for current GCP transit_external_device_conn resources?

I'm not sure if this is the best solution, maybe we should consult with backend and see why does transit_external_device_conn vpc_id require the GCP project_name and see if we can make it more uniform across resources.